package.json

```
@@ -24,10 +24,10 @@
     "build-ts": "tsc --build",
     "build-clean": "tsc --build --clean && rm -rf 'packages/**/src/{dist,esm,bundle}' && lerna run build-clean --parallel",
     "build-demo": "lerna run build-demo",
-    "test": "yarn run lint && yarn run check && yarn run build && yarn run testonly && yarn run e2e",
+    "test": "yarn run lint && yarn run check && yarn run build && yarn run testonly && lerna run build-bundles && yarn run e2e",
     "testonly": "jest && lerna run test --scope codemirror-graphql",
     "e2e": "cd packages/graphiql && yarn e2e",
-    "cypress-open": "cd packages/graphiql && yarn e2e:server & cypress open --project ./packages/graphiql",
+    "cypress-open": "cd packages/graphiql && yarn cypress-open",
```

Cleaner to use the [yarn workspace](https://yarnpkg.com/lang/en/docs/cli/workspace/) command:

```suggestion
    "cypress-open": "yarn workspace graphiql cypress-open",
```

(Untested.)

---

works a charm!